pkg/apis/azure/v1alpha1/types_infrastructure.go

```diff
@@ -165,6 +165,8 @@ type VNetStatus struct {
 type NatGatewayConfig struct {
 	// Enabled is an indicator if NAT gateway should be deployed.
 	Enabled bool `json:"enabled"`
+	// IdleConnectionTimeoutMinutes specifies the idle connection timeout limit for NAT gateway in minutes
+	IdleConnectionTimeoutMinutes *int32 `json:"idleConnectionTimeoutMinutes"`
```

Comment:
Since it's optional.

```suggestion
      // +optional
	IdleConnectionTimeoutMinutes *int32 `json:"idleConnectionTimeoutMinutes,omitempty"`
```